### ISSUE TITLE
Is this an error in the ged?

### ISSUE THREAD

**Issue by valdr** [nickreynke/python-gedcom#4]
Hello,

I'm trying to use this module, 

>>> from gedcom import Gedcom
>>> file_path = '/home/bokkie/Dropbox/Documents/Genealogie/1776\ Ballast.ged'
>>> f = Gedcom(file_path)

but I get the following trace:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/bokkie/.local/lib/python3.5/site-packages/gedcom/__init__.py", line 148, in __init__
    self.__parse(file_path)
  File "/home/bokkie/.local/lib/python3.5/site-packages/gedcom/__init__.py", line 224, in __parse
    last_element = self.__parse_line(line_number, line.decode('utf-8'), last_element)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xe9 in position 9: invalid continuation byte
```

Is there an issue with my gedcom file?

**Comment by nickreynke** [comment]
Hey @valdr! Thank you for your issue. You may want to test this again with the newest version of the package since it should be resolved now (`v0.2.2dev`). See #5 and #3.

**Comment by nickreynke** [comment]
Hey @valdr! Have you tried it again? :)
